### PR TITLE
Add Engine And Queue Configuration Options For Lens

### PR DESCRIPTION
**Pull request by bonedaddy**
Allow specifying settings for engine, and queue configurations for Lens

## todo after merge:

release new version

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -185,6 +185,15 @@ type Lens struct {
 		KeyFile  string `json:"key_file"`
 	} `json:"tls"`
 	AuthKey string `json:"auth_key"`
+	Options struct {
+		Queue struct {
+			Rate  string `json:"rate"`
```
Comment (bobheadxi) [inline types.go]:
```suggestion
			Rate  int `json:"rate"`
```

File: types.go [hunk 2]
```diff
@@ -185,6 +185,15 @@ type Lens struct {
...
+			Batch string `json:"batch"`
```
Comment (bobheadxi) [inline types.go]:
```suggestion
			Batch int `json:"batch"`
```

File: types.go [hunk 3]
```diff
@@ -185,6 +185,15 @@ type Lens struct {
 		KeyFile  string `json:"key_file"`
 	} `json:"tls"`
 	AuthKey string `json:"auth_key"`
+	Options struct {
+		Queue struct {
```
Comment (bobheadxi) [inline types.go]: `Queue` is technically a set of sub-options of `Engine`

Comment (bonedaddy) [reply types.go]: Ah true

**Review by bobheadxi — APPROVED** [review]
🔍

**Pull request merged**